learn/src/main/res/values/dimens.xml

```diff
@@ -29,9 +29,12 @@
     <dimen name="dimen_100_dp">100dp</dimen>
     <dimen name="dimen_slider_layout">100dp</dimen>
     <dimen name="dimen_youtube_view_height">250dp</dimen>
-    <dimen name="dimen_code_play_button_height">80dp</dimen>
+    <dimen name="dimen_code_play_button_height">120dp</dimen>
+    <dimen name="dimen_code_play_button_width">40dp</dimen>
```

Comment:
do we need to hardcode buttons width? Might break if we introduce multiple languages